jsonnet/components/kube-state-metrics.libsonnet

```
@@ -204,6 +234,7 @@ function(params)
                           ^kube_.+_annotations$
                         |||,
                         '--metric-labels-allowlist=pods=[*],nodes=[*],namespaces=[*],persistentvolumes=[*],persistentvolumeclaims=[*],poddisruptionbudgets=[*]',
+                        '--custom-resource-state-config-file=' + crsAssetPath,
```

From my POV, hardcoding the paths is fine and more readable. Any typo/misconfiguration will be detected by the e2e tests.

```suggestion
                        '--custom-resource-state-config-file=/etc/kube-state-metrics/custom-resource-state-config.yaml',
```